lib/assets.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+const urlJoin = require('url-join');
+const path = require('path');
+
+const dynamicRequire = require('./utils/dynamic-require');
+
+// Contains all assets that could be needed by index.html
+// Note: all asset paths are relative to host
+class Assets {
+  constructor({ urc, cssOutput, webpackAssets }) {
+    this.urc = urc;
+    this._cssOutput = cssOutput;
+    this._webpackAssets = webpackAssets;
+  }
+
+  get webpack() {
```

Comment:
This getter name is unclear to me, because you aren't actually getting Webpack, or an instance of Webpack. Could we add more detail to it?

Reply:
My intent was to use it with `assets.webpack` and hence it would be clear that we are accessing Webpack assets.
I can either make it `webpackAssets` or change it to a method named `getWebpackAssets`. I prefer the later. What do you think?

Reply:
I agree with you 👍